src/pages/joonyoung/Main/Feed/FeedCarousel/FeedCarousel.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect, useRef } from 'react';
+import {
+  BsFillArrowLeftCircleFill,
+  BsFillArrowRightCircleFill,
+} from 'react-icons/bs';
+import './FeedCarousel.scss';
+
+const FeedCarousel = ({ carouselImages }) => {
+  const [currIdx, setCurrIdx] = useState(0);
+  const carouselRef = useRef(null);
+
+  const hadnleSlideLeft = () => {
+    if (currIdx === 0) return;
+    setCurrIdx(prevIdx => --prevIdx);
+  };
+
+  const handleSlideRight = () => {
+    if (currIdx === 4) return;
+    setCurrIdx(prevIdx => ++prevIdx);
+  };
```

Comment:
index가 마지막일 때 리턴을 해주시는 것 같네요! 숫자 4의 의미가 드러날 수 있도록 변수에 할당에 활용하는 것도 방법입니다.

```suggestion
  const handleSlideRight = () => {
    const lastCardIndexNumber = 4;
    if (currIdx === lastCardIndexNumber) return;
    setCurrIdx(prevIdx => ++prevIdx);
  };
```

Reply:
!!!! 팁 감사합니다!!